src/Microsoft.DotNet.Interactive.ExtensionLab/DataExplorer.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.Formatting;
+
+namespace Microsoft.DotNet.Interactive.ExtensionLab
+{
+    public class DataExplorer
+    {
+        public DataExplorer(TabularJsonString source)
+        {
+            Id = Guid.NewGuid().ToString("N");
+            TabularData = source;
+        }
+
+        public Task LoadDataAsync(TabularJsonString source)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<TabularJsonString> GetFilteredDataAsync()
+        {
+            throw new NotImplementedException();
+        }
+
+        public string Id { get; }
+
+        public TabularJsonString TabularData { get; }
+    }
+
+    public class SetDataExplorerData : KernelCommand
```

Comment:
Why is this different from the existing display command?

Reply:
the goal here is to handle this command in the clientside kernel to just replace the data via api instead of re-display